src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs

```diff
@@ -164,7 +164,7 @@ public void GetExecutablePath_should_find_existing_executable()
             [Fact]
             public void GetExecutablePath_should_find_existing_executable_with_extension()
             {
-                FileSystem.get_executable_path("cmd.exe").ShouldEqual(
+                FileSystem.get_executable_path("cmd.exe").ToLower().ShouldEqual(

```

Comment:
Ah, appologies, copy-paste strikes again, although I thought I reverted my
unit-test touches...
Hm, there are two copies of those tests and I guess
chocolatey.tests.infrastructure.filesystem aren't being executed?

On Tue, Jan 5, 2016 at 3:07 PM, Rob Reynolds <notifications@github.com>
wrote:

> In src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs
> <https://github.com/chocolatey/choco/pull/481#discussion_r48889951>:
>
> > @@ -164,7 +164,7 @@ public void GetExecutablePath_should_find_existing_executable()
> >              [Fact]
> >              public void GetExecutablePath_should_find_existing_executable_with_extension()
> >              {
> > -                FileSystem.get_executable_path("cmd.exe").ShouldEqual(
> > +                FileSystem.get_executable_path("cmd.exe").ToLower().ShouldEqual(
>
> It's more weird in that you are shifting it to lower and asking if it is
> equal to a string with upppercase.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chocolatey/choco/pull/481/files#r48889951>.
>



-- 
*Eugene Tolmachev*

Principal Architect

*Prolucid Technologies Inc.*

*190 Robert Speck Parkway, Suite 101 | Mississauga, ON, L4Z 3K3*

*T*: 416.840.6634 | *F*: 416.342.1759 | prolucid.ca
<http://www.prolucid.ca/>


STRICTLY PERSONAL AND CONFIDENTIAL. This email may contain confidential and
proprietary material for the sole use of the intended recipient. Any review
or distribution by others is strictly prohibited. If you are not the
intended recipient please contact the sender and delete all copies.